src/libraries/System.Reflection.Context/ref/System.Reflection.Context.csproj

```diff
@@ -1,9 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <!-- this assembly is inbox in desktop, do not version it unless you
-         plan on shipping a new desktop version out of band. Instead add API
-         to a different assembly. -->
-    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
Removed as the library's Directory.Build.props already pins the AssemblyVersion.

Reply:
As you mentioned above this one was pinned to `4.0.3.0` in the Directory.Build.props. So this will mean that project references will now see the higher versioned reference.  That could cause a problem if they were loaded on .NETFramework and for whatever reason unification wasn't working (design time). 

The only project that I see that did this was `System.ComponentModel.Composition.Registration.csproj`.  That doesn't ship a copy on .NETFramework, so it's likely not a problem.

Reply:
Resolving this to have the conversation in the other thread https://github.com/dotnet/runtime/pull/84079#discussion_r1152273413